pkg/pillar/cmd/zedrouter/acl.go

```diff
@@ -215,23 +215,22 @@ func compileOldAppInstanceIpsets(ctx *zedrouterContext,
 // For a shared bridge call aclToRules for each ifname, then aclDropRules,
 // then concat all the rules and pass to applyACLrules
 // Note that only bridgeName is set with ifMgmt
-func createACLConfiglet(bridgeName string, vifName string, isMgmt bool,
-	ACLs []types.ACE, bridgeIP string, appIP string) error {
-
-	log.Infof("createACLConfiglet: ifname %s, vifName %s, ACLs %v, IP %s/%s\n",
-		bridgeName, vifName, ACLs, bridgeIP, appIP)
-	ipVer := determineIpVer(isMgmt, bridgeIP)
-	rules, err := aclToRules(bridgeName, vifName, ACLs, ipVer,
-		bridgeIP, appIP)
+func createACLConfiglet(appNetInfo types.AppNetworkInfo, ACLs []types.ACE) error {
+
+	log.Infof("createACLConfiglet: ifname %s, vifName %s, IP %s/%s, ACLs %v\n",
+		appNetInfo.BridgeName, appNetInfo.VifName, appNetInfo.BridgeIP, appNetInfo.AppIP, ACLs)
+	appNetInfo.IpVer = determineIpVer(appNetInfo.IsMgmt, appNetInfo.BridgeIP)
+	rules, err := aclToRules(appNetInfo, ACLs)
 	if err != nil {
 		return err
 	}
-	dropRules, err := aclDropRules(bridgeName, vifName)
+	dropRules, err := aclDropRules(appNetInfo.BridgeName, appNetInfo.VifName)
 	if err != nil {
 		return err
 	}
 	rules = append(rules, dropRules...)
-	return applyACLRules(rules, bridgeName, vifName, isMgmt, ipVer, appIP)
+	return applyACLRules(rules, appNetInfo.BridgeName, appNetInfo.VifName,
```

Comment:
Would it make sense to pass all of appNetInfo here?

Reply:
sure, will do the necessary changes

Reply:
I don't see those changes.